ext4: undo the stable boundary patch changes

From: Aneesh Kumar K.V <aneesh.kumar@linux.vnet.ibm.com>

This helps in applying the series with different type of
tools that expect a code diff to apply any patch.

Signed-off-by: Aneesh Kumar K.V <aneesh.kumar@linux.vnet.ibm.com>
---

 fs/ext4/extents.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)


diff --git a/fs/ext4/extents.c b/fs/ext4/extents.c
index 7ac652b..2ea7ef4 100644
--- a/fs/ext4/extents.c
+++ b/fs/ext4/extents.c
@@ -27,7 +27,6 @@
  *   - ext4*_error() should be used in some situations
  *   - analyze all BUG()/BUG_ON(), use -EIO where appropriate
  *   - smart tree reduction
- *    stable boundary change
  */
 
 #include <linux/module.h>
